Packages/CommonUI/CommonUI/Classes/View/MessageView/MessageBubleView.swift

```diff
@@ -315,11 +315,11 @@ private extension MessageBubbleView {
 					.foregroundColor(foregroundText)
 			}
 		}
-		.background(commonUIConfig.colorSet.grey2)
+        .background(isMine ? commonUIConfig.colorSet.grey2 : commonUIConfig.colorSet.primaryDefault)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/0b09249675dff5b0a460b69f27e9f3e1d518728a/Packages/CommonUI/CommonUI/Classes/View/MessageView/MessageBubleView.swift#L318"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>MessageBubleView.swift:318</code></td>
    </tr>
  </tbody>
</table>